config/main.py

```diff
@@ -260,21 +258,6 @@ def execute_systemctl(list_of_services, action):
                     if e.is_set():
                         sys.exit(1)
 
-def run_command(command, display_cmd=False, ignore_error=False):
```

Comment:
moved to utilities_common

Reply:
This was a big one on my list. It's defined in multiple locations.